101-servicebus-create-namespace-geo-recoveryconfiguration/azuredeploy.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "serviceBusNamespaceNamePrimary": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of Service Bus namespace"
+      }
+    },
+    "serviceBusNamespaceNameSecondary": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of Service Bus namespace"
+      }
+    },
+    "aliasName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of Geo-Recovery Configuration Alias "
+      }
+    },
+	"locationSecondaryNamepsace" : {
+		"type": "string",
+		"metadata": {
+			"description": "Location of Secondary namespace"
+		}
+	}
+  },
+  "variables": {
+    "defaultSASKeyName": "RootManageSharedAccessKey",
+    "defaultAuthRuleResourceId": "[resourceId('Microsoft.ServiceBus/namespaces/authorizationRules', parameters('serviceBusNamespaceNamePrimary'), variables('defaultSASKeyName'))]"
+  },
+  "resources": [
+    {
+      "apiVersion": "2017-04-01",
+      "name": "[parameters('serviceBusNamespaceNameSecondary')]",
+      "type": "Microsoft.ServiceBus/Namespaces",
+      "location": "[resourceGroup().location]",
+      "sku": {
+        "name": "Premium",
+        "tier": "Premium",
+        "capacity": 4
+      },
+      "tags": {
+        "tag1": "value1",
+        "tag2": "value2"
+      }
+    },
+    {
+      "apiVersion": "2017-04-01",
+      "type": "Microsoft.ServiceBus/Namespaces",
+	  "dependsOn": [ "[concat('Microsoft.ServiceBus/namespaces/', parameters('serviceBusNamespaceNameSecondary'))]" ],
+      "name": "[parameters('serviceBusNamespaceNamePrimary')]",
+      "location": "South Central US",
```

Comment:
Since this is a geo-dist sample add location parameters for each location

Reply:
resolved